helm/cluster-operator-chart/templates/rbac.yaml

```diff
@@ -22,6 +22,19 @@ rules:
       - update
       - delete
       - get
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+      - networkpolicies
```

Comment:
Hey Vol, I see network policies in different apiGroup in docs, why?

https://kubernetes.io/docs/concepts/services-networking/network-policies/

Reply:
Eh, true

Reply:
But I tested that with draughtsman and it worked fine https://github.com/giantswarm/draughtsman/pull/83/files

Reply:
That is confusing but if this is working let's go with it. It's important to align the RBAC between components because I don't want them to fight.